pdal_features.hpp.in

```diff
@@ -6,17 +6,21 @@
  */
 #pragma once
 
+#include <string>
+
 /*
  * version settings
  */
-#define PDAL_VERSION_MAJOR @PDAL_VERSION_MAJOR@
-#define PDAL_VERSION_MINOR @PDAL_VERSION_MINOR@
-#define PDAL_VERSION_PATCH @PDAL_VERSION_PATCH@
+namespace pdal
```

Comment:
this has the potential to break a bunch of stuff. External users check these ifdefs to test versions at compile time. Is there a reason to ditch them?